functions/src/SPL/index.ts

```diff
@@ -1,6 +1,7 @@
 import * as functions from 'firebase-functions';
```

Comment:
Same comments as in the other file, regarding naming, casing, truthiness checks, and error handling